LabApiUtilities/src/test/com/microsoft/identity/labapi/utilties/authentication/common/MicrosoftClientAssertionTest.java

```diff
@@ -42,6 +43,7 @@ public class MicrosoftClientAssertionTest {
     private final static String KEYSTORE_PROVIDER = "SunMSCAPI";
     private final static String AUDIENCE = "some_audience";
 
+    @Ignore
```

Comment:
I believe this test is independent of the changes you are making and test likely still valid

Reply:
reverted